smk-create/template-default/resources/assets/style.css

```diff
@@ -12,7 +17,11 @@ body {
     justify-content: stretch;
     align-items: stretch;
 
-    font-family: Helvetica,Arial,sans-serif;
+    font-family: 'BC Sans',Helvetica,Arial,sans-serif;
+}
+
+button, input {
+    font-family: 'BC Sans',Helvetica,Arial,sans-serif;
```

Comment:
Consider setting the font-family for buttons to 'inherit' in order to make updating the app wide font a little easier for users.

font-family: inherit

Reply:
Done.